spec/Spec/BlsEC.cry

```diff
@@ -0,0 +1,226 @@
+/**
+ * This module contains functions that perform ECC Point operations.
+ * These functions are specific to the BLS curve, and specifially G1.
+ *
+ * The Deneb Python specs use the py_ecc library for the same operations.
+ * This modules functions are based off of py_ecc.
+ * @see https://github.com/ethereum/py_ecc/blob/main/py_ecc/bls12_381/bls12_381_curve.py
+ */
+module Spec::BlsEC where
+
+import Common::Utils
+
+/**
+ * Size of a G1 point
+ */
+type G1Bits = 384
+
+/**
+ * Serialized G1 point
+ */
+type Bytes48 = [G1Bits]
+
+/**
+ * G1 elliptic curve point
+ */
+type G1Point =
+    { xCoord : Bytes48
+    , yCoord : Bytes48
+    }
```

Comment:
How do you represent the point at infinity? It doesn't have a set of affine coordinates but there are often conditionals in various algorithms that require checking for it. For the [prime field elliptic curves](https://github.com/GaloisInc/cryptol-specs/blob/master/Common/EC/PrimeField/PFEC.cry#L140) I used an enum but that is [not supported by SAW yet](https://github.com/GaloisInc/saw-script/issues/2052) so might not be an option for this project.

Reply:
I'm using (0, 0) for infinity --- I think that gets added in a later commit.

Reply:
As in my next PR.